aw/ViewControllers/Reaches/Detail/ReachDetailMapViewController.swift

```diff
@@ -29,13 +29,17 @@ class ReachDetailMapViewController: UIViewController {
     }
 
     deinit {
-        mapView.delegate = nil
+        if let mapView = mapView {
+            mapView.delegate = nil
+        }
     }
 }
 
 // MARK: - Extension
 extension ReachDetailMapViewController {
     func initialize() {
+        guard let mapView = mapView else { return }
```

Comment:
Why the use of `guard` here and below, but `if let` elsewhere, and in the other map file? Might be nice to be consistent.

Reply:
Cause my brain didn't quite catch up with the transition from PHP > Python > JavaScript > Swift in the last 24 hours fast enough to remember `guard` initially. I'm keeping a few `if lets` around where things do need to happen afterwards.